### ISSUE TITLE
Why, in version v8.2.0 does the box point to >=7.0.0?

### ISSUE THREAD

**Issue by petemcfarlane** [laravel/homestead#1081]
I'm trying to install the latest virtual machine box, I've checked out v8.2.0, I've tried overriding the `name: "homestead-8"` and `version: ">= v8.2.0` in my Homestead.yml file,

But when I run `vagrant up` I can't find the box anywhere

I can't see a box on vagrant cloud, with version 8.2 - what am I missing?

Also, the Homestead.rb file is still referencing `homestead-7` as a name and the version is `>= 7.0.0`
https://github.com/laravel/homestead/blob/68388101a1364019c1fa44ff427be43b5ffda90f/scripts/homestead.rb#L19-L21

## Versions
  - Vagrant: 2.2.4
  - Provider: Virtualbox 6.0
  - Homestead: 2.2.2

## Host operating system
Mac OS X

## Homestead.yaml
```yaml
---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox
version: ">= 8.2.0"
name: "homestead-8"

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: ~/Sites
      to: /home/vagrant/sites

sites:
    - map: example.text
      to: /home/vagrant/sites/example/public

databases:
    - homestead
```

## Vagrant destroy & up output
```
==> homestead-8: VM not created. Moving on...
Bringing machine 'homestead-8' up with 'virtualbox' provider...
==> homestead-8: Box 'laravel/homestead' could not be found. Attempting to find and install...
    homestead-8: Box Provider: virtualbox
    homestead-8: Box Version: >= 8.2.0
==> homestead-8: Loading metadata for box 'laravel/homestead'
    homestead-8: URL: https://vagrantcloud.com/laravel/homestead
The box you're attempting to add has no available version that
matches the constraints you requested. Please double-check your
settings. Also verify that if you specified version constraints,
that the provider you wish to use is available for these constraints.

Box: laravel/homestead
Address: https://vagrantcloud.com/laravel/homestead
Constraints: >= 8.2.0
Available versions: 0.1.0, 0.1.1, 0.1.2, 0.1.3, 0.1.4, 0.1.5, 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.2.0, 0.2.1, 0.2.2, 0.2.4, 0.2.5, 0.2.6, 0.2.7, 0.3.0, 0.3.3, 0.4.0, 0.4.1, 0.4.2, 0.4.4, 0.5.0, 0.6.0, 0.6.1, 1.0.0, 1.0.1, 1.1.0, 2.0.0, 2.1.0, 2.2.0, 3.0.0, 3.1.0, 4.0.0, 5.0.1, 5.1.0, 5.2.0, 6.0.0, 6.1.0, 6.2.0, 6.3.0, 6.4.0, 7.0.0, 7.1.0
```

## Expected behavior
Expect to run the latest version of Homestead

## Actual behavior
Can't find the box anywhere

**Comment by deviantintegral** [comment]
I think this is because the base box is actually laravel/settler, which is last released as 7.1.0: https://github.com/laravel/settler/releases

On the one hand, separate projects should be able to have independent versions, but perhaps it's simpler to just tag both repositories at the same time for corresponding releases.

**Comment by petemcfarlane** [comment]
Ah thank you, so I am indeed running the latest version already. It somewhat confused me, but thank you for explaining.

**Comment by svpernova09** [comment]
Since both repos follow semver they will never be the same version. The constraint you see is how we ensure you can’t easily use an incompatible base box with an incompatible repo version.

--
- Joe Ferguson
JoeFerguson.me
osmihelp.org

**Comment by petemcfarlane** [comment]
it's also confusing as when I start the `homestead-7` box the first thing I see is 
```
➜  Homestead git:(6838810) vagrant ssh
Welcome to Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-45-generic x86_64)

 _                               _                 _ 
| |                             | |               | |
| |__   ___  _ __ ___   ___  ___| |_ ___  __ _  __| |
| '_ \ / _ \| '_ ` _ \ / _ \/ __| __/ _ \/ _` |/ _` |
| | | | (_) | | | | | |  __/\__ \ ||  __/ (_| | (_| |
|_| |_|\___/|_| |_| |_|\___||___/\__\___|\__,_|\__,_|

* Homestead 8.0.0 released! PHP 7.3 is now the default! 
* Settler v7.0.0 released! Make sure you update
* Need PHP 5.6 or 7.0? Homestead 7.x Settler 6.4.0

0 packages can be updated.
0 updates are security updates.

```

Which sounds like I should update to homestead 8.0.0!

**Comment by svpernova09** [comment]
The 7 in homestead-7 isn’t a version number. You’re certainly not the first. Maybe it’s time to drop that -7

--
- Joe Ferguson
JoeFerguson.me
osmihelp.org
